### PR TITLE
Add instructions on IP config for UISO

### PR DESCRIPTION
**Pull request by agrawalkaushik**

### CODE REVIEW

File: docs/initialize_u-iso_adv_workflow.md [hunk 1]
```diff
@@ -21,9 +21,16 @@ Select the initialization workflow that suits the needs of your deployment.
 - [File on Disk](#file-on-disk)
 - [API Initialization](#api-initialization)
 
+## Connection Setup
+For non console access you can connect your laptop or other device to any of the designated LAN port and connect to 192.168.128.1 to connect to SSR PCLI and/or shell.
```
Comment (MichaelBaj) [inline docs/initialize_u-iso_adv_workflow.md:25]: The other document specifies connecting to http://192.168.128.1. Should this be the same?

File: docs/initialize_u-iso_device.md [hunk 2]
```diff
@@ -3,7 +3,15 @@ title: Initialize Your Device - Web Workflow
 sidebar_label: Initialize Your Device - Web Workflow
 ---
 
-This is the part where configuring your device gets really easy! Using your browser, choose the workflow below that best suits your needs: 
+This is the part where configuring your device gets really easy!
+
+As a first step, connect your laptop or other device to any of the designated LAN port and connect to http://192.168.128.1 to connect to SSR onboarding UI
```
Comment (MichaelBaj) [inline docs/initialize_u-iso_device.md:8]: you will have to add the directives:




Around this line as the broken link checker is going to try to validate http://192.168.128.1.

You received the following build error:
[✖] http://192.168.128.1/ → Status: 0 Error: ETIMED

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/initialize_u-iso_adv_workflow.md (hunk 1 above)
Comment (agrawalkaushik) [reply docs/initialize_u-iso_adv_workflow.md:25]: The other document is for web but this one is for PCLI and API based workflow.

File: docs/initialize_u-iso_device.md (hunk 2 above)
Comment (agrawalkaushik) [reply docs/initialize_u-iso_device.md:8]: bummer - I was hoping to keep it a hyperlink. Is there a special directive for this to allow both i.e. skip the check but keep it a hyperlink?

File: docs/initialize_u-iso_adv_workflow.md (hunk 1 above)
Comment (MichaelBaj) [reply docs/initialize_u-iso_adv_workflow.md:25]: Perhaps change the wording then to state SSH to 192.168.128.1 to be more explicit?

File: docs/initialize_u-iso_device.md (hunk 2 above)
Comment (MichaelBaj) [reply docs/initialize_u-iso_device.md:8]: Adding the directives (which were removed from the comments....) will preserve the hyperlink,

Comment (agrawalkaushik) [reply docs/initialize_u-iso_device.md]: Thanks - I was able to see it now!

File: docs/initialize_u-iso_adv_workflow.md (hunk 1 above)
Comment (agrawalkaushik) [reply docs/initialize_u-iso_adv_workflow.md:25]: the same address can be used for API calls to FWIW but I'll add some language to be clear.

**Review by MichaelBaj — APPROVED** [review]

File: docs/initialize_u-iso_device.md [hunk 3]
```diff
@@ -3,7 +3,17 @@ title: Initialize Your Device - Web Workflow
 sidebar_label: Initialize Your Device - Web Workflow
 ---
 
-This is the part where configuring your device gets really easy! Using your browser, choose the workflow below that best suits your needs: 
+This is the part where configuring your device gets really easy!
+
+<!-- markdown-link-check-disable -->
+As a first step, connect your laptop or other device to any of the designated LAN port and connect to https://192.168.128.1 to access the SSR onboarding UI
```
Comment (Chr1st0ph3rTurn3r) [inline docs/initialize_u-iso_device.md:9]:
```suggestion
As a first step, connect your laptop or other device to any of the designated LAN ports and connect to https://192.168.128.1 to access the SSR onboarding UI.
```

File: docs/initialize_u-iso_device.md [hunk 4]
```diff
@@ -3,7 +3,17 @@ title: Initialize Your Device - Web Workflow
...
+<!-- markdown-link-check-enable -->
+
+:::note
+The laptop or other client device must be assigned a static IP address within the range of `192.168.128.2` to `192.168.128.254` along with a subnet mask of `/24` or `255.255.255.0`
```
Comment (Chr1st0ph3rTurn3r) [inline docs/initialize_u-iso_device.md:13]:
```suggestion
The laptop or other client device must be assigned a static IP address within the range of `192.168.128.2` to `192.168.128.254` along with a subnet mask of `/24` or `255.255.255.0`.
```

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Pull request merged**